tests/integration/sdk/test_scraping.py

```diff
@@ -25,6 +26,13 @@ async def test_scraping_markdown():
         assert data.markdown is not None
 
 
+@pytest.mark.asyncio
+async def test_pdf_scraping_markdown():
```

Comment:
can you add a test with a real agent workflow ? I.e one task could be to extract the first paragraph of the "Attention is all you need" paper starting from arxiv.com. I want to make sure that the flow is correct end to end. I suspect that it's not because observe could fail once the agent reaches the pdf page but lets see